Cargo.lock

```diff
@@ -2727,9 +2604,9 @@ dependencies = [
 
 [[package]]
 name = "num-traits"
-version = "0.2.15"
+version = "0.2.14"
```

Comment:
Why did you downgrade to 0.2.14?

Reply:
> > > @dmakarov I still see a number of tokio crates being upgraded, can you comment on whether those are necessary or can be downgraded as well?
> > 
> > 
> > tokio was updated on master recently
> 
> I couldn't find that commit, can you point me to it?

my mistake. I assumed it was based on previous comments in this PR.